build.sbt

```diff
@@ -95,7 +96,8 @@ coverageMinimumStmtTotal := 91
 coverageFailOnMinimum := true
 coverageExcludedPackages := "<empty>;com.kenshoo.play.metrics.*;prod.*;testOnlyDoNotUseInAppConf.*;app.*;uk.gov.hmrc.BuildInfo;"
 
-ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-java8-compat" % "always"
+ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml"          % VersionScheme.Always
```

Comment:
This can be updated to  `libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always`